AxonDeepSeg/segment.py

```diff
@@ -179,9 +147,11 @@ def segment_folders(path_testing_images_folder, path_model,
 							  resampled_resolutions=resolution_model, prediction_proba_activate=False,
 							  write_mode=True)
 
-			if verbosity_level >= 1:
-				print "Image {0} segmented.".format(os.path.join(path_testing_images_folder, file_))
-	# The segmentation has been done for this image folder, we go to the next one.
+		if verbosity_level >= 1:
+			print "Image {0} segmented.".format(os.path.join(path_testing_images_folder, file_))
```

Comment:
If this is inside the `tqdm` loop, use `tqdm.write()` instead.

Reply:
Yes, corrected.